lib/src/Uncertainty/Bayesian/GaussianNonLinearCalibration.cxx

```diff
@@ -334,7 +333,7 @@ void GaussianNonLinearCalibration::run()
   const TriangularMatrix errorInverseCholesky(error.getInverseCholesky());
   const Point thetaStar(run(inputObservations_, outputObservations_, getCandidate(), parameterInverseCholesky, errorInverseCholesky));
   // Build the residual function this way to benefit from the automatic Hessian
-  const MemoizeFunction residualFunction(NonLinearLeastSquaresCalibration::BuildResidualFunction(model_, inputObservations_, outputObservations_));
+  const Function residualFunction(NonLinearLeastSquaresCalibration::BuildResidualFunction(model_, inputObservations_, outputObservations_));
```

Comment:
So for this algorithm and the other calibration algorithms, it is the user's responsibility to activate a cache on his function. Does it allow to reuse the cache if the algorithm is run twice?

Reply:
yes it should